internal/controller/semester.controller.go

```diff
@@ -189,3 +189,31 @@ func (sc *SemesterController) AdminDeleteSemester(ctx *gin.Context) {
 
 	response.SuccessResponse(ctx, http.StatusOK, dto.OutputCommon{Message: message})
 }
+
+// @Summary GetListSemestersWithCountGroup
+// @Description Get List Semesters With Count Group
+// @Tags Semester
+// @Produce json
+// @Param limit query int true "Limit"
+// @Param page query int true "Page"
+// @Router /semesters/count  [get]
```

Comment:
this should name semester/statistics instead count
Best practice naming router in restful api https://restfulapi.net/resource-naming/